actions/bcr-pr-review-notifier/action.yml

```diff
@@ -0,0 +1,10 @@
+name: 'BCR PR Review Notifier'
+description: 'Helps notify people for reviewing PRs of Bazel Central Registry.'
+inputs:
+  token:
+    description: 'The GitHub token'
+    required: false
+    default: ${{ github.token }}
+runs:
+  using: 'node16'
```

Comment:
node16 is deprecated in favor of node20 (https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/)

Reply:
Thanks!